### ISSUE TITLE
Keep track of verbatim string in Untyped Syntax Tree.

### ISSUE THREAD

**Issue by nojaf** [dotnet/fsharp#10209]
**Is your feature request related to a problem? Please describe.**
Related to Fantomas bug https://github.com/fsprojects/fantomas/issues/560.
In short, the AST does not keep track of the `@` character before a string.

```fsharp
let s = @"\"
```

Creates

```fsharp
SynConst.String("\", tmp.fsx (1,8--1,12) IsSynthetic=false), tmp.fsx (1,8--1,12) IsSynthetic=false)
```

**Describe the solution you'd like**
Can we extend `SynConst.String` to indicate the leading `@` was present?

**Describe alternatives you've considered**
In Fantomas, we currently detect the `@` via the tokens.

**Additional context**
If approved, I'm willing to try and implement this myself.

**Comment by nojaf** [comment]
Hello @cartermp, I would be interested in an FCS release because of this. Do I need to raise this somewhere formally?

**Comment by cartermp** [comment]
We don't have a good process in place yet. We're closed down on the VS 16.9 release, which will correspond to FCS 39 - at least for now - but do have several ways to work with that. We (myself, @KevinRansom, @dsyme, etc.) need to establish what the policy for releases will be though. The past isn't a great guide since there could be releases a week apart or 6 months apart. But we do think that only following the schedule of the F# tools for VS isn't sufficient either.
